include/envoy/server/options.h

```diff
@@ -66,9 +66,16 @@ class Options {
 
   /**
    * @return const std::string& the path to the configuration file.
+   *         If this value is set configJson() will be empty.
    */
   virtual const std::string& configPath() PURE;
 
+  /**
+   * @return const std::string& of the full configuration as json.
+   *         If this value is set configPath() will be empty.
+   */
+  virtual const std::string& configJson() PURE;
```

Comment:
Can you add user docs for this flag and release notes in a parallel PR in https://github.com/envoyproxy/data-plane-api?

Reply:
Could not find the release notes, but got this started: https://github.com/envoyproxy/data-plane-api/pull/577

Reply:
Found the release notes; updated that PR.